.github/workflows/remoteconfig.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     strategy:
       matrix:
         target: [ios, tvos, macos, watchos]
-        podspec: [FirebaseRemoteConfig.podspec, FirebaseRemoteConfigSwift.podspec --skip-tests]
+        podspec: [FirebaseRemoteConfig.podspec, FirebaseRemoteConfigSwift.podspec --allow-warnings --skip-tests]
```

Comment:
Just for my own understanding, what are the new warnings that are produced?

Reply:
```
All of the public API from `FirebaseRemoteConfigSwift` can now be accessed through 
the `FirebaseRemoteConfig` module. Therefore, the `FirebaseRemoteConfigSwift` 
module is deprecated and will be removed in the future. See 
https://firebase.google.com/docs/ios/swift-migration for migration instructions.
```

Reply:
This is declared in the diff in `FirebaseRemoteConfigSwift/Sources/FirebaseRemoteConfigSwift.swift`.

Reply:
Thanks for clarifying! Somehow it didn't click in my head that it was that `#warning`.